examples/common/scripts/vtadmin-up.sh

```diff
@@ -47,6 +47,28 @@ done
 # Check one last time
 [[ $(curl -s "http://localhost:${vtadmin_api_port}/api/clusters") == "${expected_cluster_result}" ]] || fail "vtadmin failed to discover the running example Vitess cluster."
 
+# Download nvm and node
+output() {
+  echo -e "$@"
+}
```

Comment:
IMO we should add this to `../lib/utils.sh` which will get read by default via env.sh.